Source/WebCore/xml/XMLErrors.h

```diff
@@ -43,7 +43,7 @@ class XMLErrors {
 public:
     explicit XMLErrors(Document&);
 
-    enum ErrorType { warning, nonFatal, fatal };
+    enum class ErrorType : uint8_t { Warning, NonFatal, Fatal };
```

Comment:
```suggestion
    enum class Type : uint8_t { Warning, NonFatal, Fatal };
```

You could shorten this to Type. The outer `XMLErrors` namespace already provides context